cypress/e2e/dresses-list.cy.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  MenuContentPage,
+  DressesListPage}
+  from "../page/index";
```

Comment:
this should be in this way:
```suggestion
  DressesListPage
}  from "../page/index";
```